fastembed/late_interaction/colbert.py

```diff
@@ -69,10 +69,15 @@ def _post_process_onnx_output(
     def _preprocess_onnx_input(
         self, onnx_input: Dict[str, np.ndarray], is_doc: bool = True
     ) -> Dict[str, np.ndarray]:
-        if is_doc:
-            onnx_input["input_ids"][:, 1] = self.DOCUMENT_MARKER_TOKEN_ID
-        else:
-            onnx_input["input_ids"][:, 1] = self.QUERY_MARKER_TOKEN_ID
+        original_length = onnx_input["input_ids"].shape[1]
+        marker_token = self.DOCUMENT_MARKER_TOKEN_ID if is_doc else self.QUERY_MARKER_TOKEN_ID
+
+        onnx_input["input_ids"] = np.insert(onnx_input["input_ids"], 1, marker_token, axis=1)
+        onnx_input["attention_mask"] = np.insert(onnx_input["attention_mask"], 1, 1, axis=1)
+
+        if not is_doc:
+            onnx_input["input_ids"] = onnx_input["input_ids"][:, :original_length]
+            onnx_input["attention_mask"] = onnx_input["attention_mask"][:, :original_length]
```

Comment:
why are we cutting back to `original_length`? I thought we should do it only if we exceed the model's context window size

Reply:
why is it expected to be 32?